NotificationTrigger/time.js

```diff
@@ -0,0 +1,12 @@
+const { NOTIFICATION_END_TIME, NOTIFICATION_START_TIME } = require('./config')
```

Comment:
@shrijan-siris may be rename time.js to verifyNoticiableTime?

Reply:
renamed as notificationTime.js so that we can have other related time functions with notification